react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+// let count = 0;
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      color: 'purple',
+      count: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  render() {
+    return (
```

Comment:
Here you can call the `getColor` method and save the color into a variable, that can then be used to set the `backgroundColor`

```JSX
const color = this.getColor();
```